ctrlf.el

```diff
@@ -212,6 +226,12 @@ See `ctrlf-split-fuzzy' for how INPUT is split into subinputs.
 The subinputs are joined with \".*\"."
   (string-join (ctrlf-split-fuzzy input) ".*"))
 
+(defun ctrlf-translate-symbol (input)
```

Comment:
Super nice side effect of this is that the symbol boundary constructs don't appear in the user's input, making it exactly like `isearch-forward-symbol`. Furthermore, we automatically get correct behavior with `ctrlf-occur`, too.